backend/src/graphql/model/Decay.ts

```diff
@@ -1,20 +1,24 @@
 import { ObjectType, Field, Int } from 'type-graphql'
 import Decimal from 'decimal.js-light'
 
+interface DecayInterface {
+  balance: Decimal
+  decay: Decimal
+  roundedDecay: Decimal
+  start: Date | null
+  end: Date | null
+  duration: number | null
+}
```

Comment:
Why did you do this? Doesn't this make the code more complex to read with zero benefit? You don't export this interface...